src/rollup/index.ts

```diff
@@ -175,99 +170,51 @@ export default function rollup(
 							timeEnd('GENERATE', 1);
 
 							const output = {
+								name: relative(process.cwd(), resolve(outputOptions.file || inputOptions.input)),
```

Comment:
I can't remember if `process.cwd()` is ok to use for browser builds or not. Would be good to check this.